### PR TITLE
Fix copying to clipboard on UI

### PR DESCRIPTION
**Pull request by wendigo**
Fixes https://github.com/prestosql/presto/issues/2782

### CODE REVIEW

**Comment by sopel39** [comment]
Did UI changed? If so, can you post screenshot?

**Comment by wendigo** [comment]
Visually not. Behaviour changed. Posting right now.

**Comment by wendigo** [comment]
Copying to clipboard now works in every place in UI:

Zrzut ekranu 2020-02-19 o 10 44 48
Zrzut ekranu 2020-02-19 o 10 45 00
Zrzut ekranu 2020-02-19 o 10 45 13

**Comment by wendigo** [comment]
Rebased on top of master

File: presto-main/etc/hive-default-fs-site.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<configuration>
```
Comment (martint) [inline presto-main/etc/hive-default-fs-site.xml]: Unrelated change?

Comment (sopel39) [reply presto-main/etc/hive-default-fs-site.xml]: remove, bad rebase

Comment (wendigo) [reply presto-main/etc/hive-default-fs-site.xml]: Sorry :)

Comment (wendigo) [reply presto-main/etc/hive-default-fs-site.xml]: Sorry :)

File: presto-main/src/main/resources/webapp/src/components/LivePlan.jsx [hunk 2]
```diff
@@ -204,6 +204,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
 
     componentDidMount() {
         this.refreshLoop.bind(this)();
+        new window.ClipboardJS('.copy-button');
```
Comment (findepi) [inline presto-main/src/main/resources/webapp/src/components/LivePlan.jsx]: technically, the `window.` is redundant. What's the rationale?

Comment (wendigo) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx]: @flow checker

Comment (findepi) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx]: so it's a "best practise". We don't use it on CI, do we?

Comment (wendigo) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx]: We do (on couple of files) :)

Here: https://github.com/prestosql/presto/blob/master/presto-main/src/main/resources/webapp/src/components/LivePlan.jsx#L14,
https://github.com/prestosql/presto/blob/master/presto-main/src/main/resources/webapp/src/components/PageTitle.jsx#L14

Comment (wendigo) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx]: Flow is run here: https://github.com/prestosql/presto/blob/master/presto-main/bin/check_webui.sh#L40
